Libraries/Components/TextInput/TextInput.js

```diff
@@ -252,6 +252,15 @@ var TextInput = React.createClass({
      */
     selectionState: PropTypes.instanceOf(DocumentSelectionState),
     /**
+     * The start and end of the text input's selection. Set start and end to
+     * the same value to position the cursor.
+     * @platform ios
+     */
+    selection: PropTypes.shape({
```

Comment:
I don't see any code that's actually making this a controlled prop - if you set it in JS, then the user changes it, what's going to change it back to the JS value? Normally this is done with setNativeProps in a change handler - see line 563 below as an example.